app/src/main/java/com/teamagam/gimelgimel/app/map/actions/send/dynamicLayers/MultiplePointsMapDrawer.java

```diff
@@ -0,0 +1,40 @@
+package com.teamagam.gimelgimel.app.map.actions.send.dynamicLayers;
+
+import com.teamagam.gimelgimel.app.map.actions.MapDrawer;
+import com.teamagam.gimelgimel.app.map.actions.MapEntityFactory;
+import com.teamagam.gimelgimel.domain.map.entities.geometries.PointGeometry;
+import com.teamagam.gimelgimel.domain.map.entities.mapEntities.GeoEntity;
+import com.teamagam.gimelgimel.domain.map.entities.mapEntities.PointEntity;
+import com.teamagam.gimelgimel.domain.map.entities.symbols.Symbol;
+import java.util.List;
+
+public abstract class MultiplePointsMapDrawer extends AbsMapDrawer {
+
+  protected MultiplePointsMapDrawer(MapEntityFactory mapEntityFactory,
+      MapDrawer mapDrawer,
+      OnEditingStartListener listener) {
+    super(mapEntityFactory, mapDrawer, listener);
+  }
+
+  protected GeoEntity buildEntity(MapEntityFactory mapEntityFactory,
+      List<PointGeometry> pointsHistory,
+      Symbol symbol) {
+    if (isOnlyOnePoint(pointsHistory)) {
+      return createFirstPoint(mapEntityFactory, pointsHistory);
+    }
+    return createEntity(mapEntityFactory, pointsHistory, symbol);
+  }
+
+  abstract GeoEntity createEntity(MapEntityFactory mapEntityFactory,
```

Comment:
createMultipointEntity

Reply:
Thinking over it again, it should be consistent with current naming - 
use build instead of create.
Both for this method and the others here.

Reply:
done